README.md

```diff
@@ -1,2 +1,71 @@
-# mobilefirstportfolio
-This is a portfolio website that has prioritized mobile user  experience over Desktop user.
+# mobilefirstportfolio website.
+
+This is a portfolio website that has prioritized mobile user  experience over Desktop user experience.
+
+## Built With
+
+- HTML and CSS.  
+- Javascript  
+
+
+## Getting Started
+This is a simple mobile first website built based on 
+[@figma design templates.](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1)
+
+To get a local copy up and running follow these simple example steps.
+
+### Prerequisites
+-Text editor    
+-Browser    
+
+### Setup
+-Clone this project using the following steps;  
+1.[@Visit the project page](https://github.com/borrnelsonk/mobilefirstportfolio)    
+2.Navigate to code section and click on clone.    
+3.[@Copy the web url](https://github.com/borrnelsonk/mobilefirstportfolio.git)    
+4.Navigate to the local path where you want to clone the repository.    
+5.Git clone the repository.  
+
+### Install
+-This project requires node modules installed in the root directory.  
+-[@Check this steps on how to install them](https://github.com/microverseinc/linters-config/tree/master/html-css)  
+
+### Run tests
+-Check your code for linter errors.  
+-Preview your website on the browser.  
+
+### Deployment.
+-[@checkout these deployment options](https://www.codecademy.com/learn/deploy-a-website)
+
+
+
+
+## Authors
+
+👤 **Bor Nelson**
+
+- GitHub: [@githubhandle](https://github.com/borrnelsonk)
+- Twitter: [@twitterhandle](https://twitter.com/bornelsonn)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/bornelson/)
+
+
+
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](https://github.com/borrnelsonk/mobilefirstportfolio/issues).
+
+## Show your support
+
+Give a ⭐️ if you like this project!
+
+## Acknowledgments
+
+- The Microverse community.
+-My standup team.
+
+
+## 📝 License
+
+This project is [MIT](./LICENSE) licensed.
```

Comment:
- [ ] As mentioned by the previous reviewers, This link is redirecting to the non-existing page. Kindly remove this section or create an MIT license file for your application.